app/views/refinery/_footer.html.erb

```diff
@@ -1,35 +1,34 @@
 <div class="container">
   <div class="footer-top">
     <div class="link-section footer-about-links">
-      <span class="link-category-heading">About MAPC</span>
+      <span class="link-category-heading"><%= t('footer.about_mapc') %></span>
       <ul>
-        <li><a href="">Plan Process</a></li>
-        <li><a href="https://www.mapc.org/about-mapc/staff/">Staff</a></li>
+        <li><a href="https://www.mapc.org/about-mapc/staff/"><%= t('footer.staff') %></a></li>
         <li><a href="https://www.mapc.org/get-involved/metrofuture-our-regional-plan/">MetroFuture</a></li>
       </ul>
     </div>
     <div class="link-section other-links">
       <div class="top-links">
-        <span class="link-category-heading">Receive Updates</span>
-        <a class="link-button" href="<%= CONSTANT_CONTACT_LANDING_PAGE %>" target="_blank">Join our email list</a>
+        <span class="link-category-heading"><%= t('footer.receive_updates') %></span>
+        <a class="link-button" href="<%= CONSTANT_CONTACT_LANDING_PAGE %>" target="_blank"><%= t('footer.join_our_email_list') %></a>
       </div>
       <div class="bottom-links">
-        <a href="/privacy">Privacy Policy</a>
+        <a href="/privacy"><%= t('footer.privacy_policy') %></a>
       </div>
     </div>
   </div>
   <div class="footer-bottom">
     <div class="footer-mapc-brand">
-      <%= image_tag("mapc_logo.svg", class: "logo") %>
-      <div class="mapc">Metropolitan Area Planning Council</div>
+      <%= image_tag("mapc_logo.svg", class: "logo", alt: t('logo_alt')) %>
+      <div class="mapc"><%= t('mapc') %></div>
```

Comment:
Are you sure we should be translating a proper noun? For example, does Apple translate Apple into Chinese or use the English version?

Reply:
Given that "Metropolitan Area Planning Council" is translatable and that the UN does translate agency names like so (http://www.un.org/fr/sections/about-un/funds-programmes-specialized-agencies-and-others/index.html) I'm inclined to translate MAPC.